.github/issue_template/proposal.md

```diff
@@ -0,0 +1,13 @@
+---
+name: 💥 Proposal
```

Comment:
How is this different from "Feature"?

Reply:
@JelleZijlstra I have deleted the `Proposal.md` file.